ibm/service/power/resource_ibm_pi_volume_group.go

```diff
@@ -76,7 +76,7 @@ func ResourceIBMPIVolumeGroup() *schema.Resource {
 				Computed:    true,
 				Description: "Volume Group Replication Status",
 			},
-			"status_description_errors": vgStatusDescriptionErrors(),
+			//"status_description_errors": vgStatusDescriptionErrors(),
```

Comment:
Reason for the comment?

Reply:
@Diptipowervs please remove this comment.

Reply:
Removed

Reply:
The full line is removed now.
Don't we need `status_description_errors`?

Reply:
so that's still there and got unrolled into 
```
Attr_StatusDescriptionErrors: {
```